packages/shared_preferences/shared_preferences/test/shared_preferences_test.dart

```diff
@@ -215,7 +215,7 @@ void main() {
   });
 }
 
-class FakeSharedPreferencesStore implements SharedPreferencesStorePlatform {
+class FakeSharedPreferencesStore extends SharedPreferencesStorePlatform {
```

Comment:
You can get rid of `isMock` since that's only needed to bypass the check that would prevent `implements`